ansible/roles/sonicv2/templates/etc/systemd/system/syncd.j2

```diff
@@ -7,11 +7,13 @@ After=database.service
 User=root
 {% if sonic_hwsku == 'ACS-MSN2700' %}
 ExecStartPre=/etc/init.d/sxdkernel start
+ExecStartPre=/usr/bin/mst start
```

Comment:
is it better to start the mst right after the system boots and keeps running without any dependency with docker syncd